assets/scss/_header.scss

```diff
@@ -0,0 +1,57 @@
+.header {
+    background-image: url('../images/header-bg-mobile.svg');
+    background-size: 100% auto;
+    background-repeat: no-repeat;
+
+    @include media-queries('tablet') {
+        background-image: url('../images/header-bg.svg');
+    }
+}
+
+.nav__list {
+    min-height: 20vw;
```

Comment:
We set here height by device width. Is it desired solution? I think, that on very large screens that header will be very tall, likewise on small devices - it will be small. You could use set `px` height, or even better - don't set height at all. Let content decide about height

Reply:
The minimum height is set to prevent cutting of the background. The vw units are used to keep the proportions of the background. To keep the ratio, the height of it depends on the width. As we discussed, to prevent from having a problem on very large screens, I will add a new breakpoint for large screens and set max values for the page.